<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDEBUG</class>
 <widget class="QFrame" name="FormDEBUG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPlainTextEdit" name="textBox1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="plainText">
    <string>Box di debug</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textBox2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnX">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHistory">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPause">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
